template.yml

```diff
@@ -206,8 +206,228 @@ Resources:
             Resource: !Sub "arn:${AWS::Partition}:iam::*:role/EmergencyAccess_RO"
       UserName: !Ref rEmergencyAccessReadOnlyUser
 
+  rFunctionLogGroup:
+    Type: "AWS::Logs::LogGroup"
+    UpdateReplacePolicy: Delete
+    DeletionPolicy: Delete
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W84
+            reason: "Ignoring KMS key"
+    Properties:
+      LogGroupName: !Sub "/aws/lambda/${rFunction}"
+      RetentionInDays: 3
+
+  rFunctionRole:
+    Type: "AWS::IAM::Role"
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W11
+            reason: "Ignoring wildcard policy"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          Effect: Allow
+          Principal:
+            Service: !Sub "lambda.${AWS::URLSuffix}"
+          Action: "sts:AssumeRole"
+          Condition:
+            StringEquals:
+              "aws:SourceAccount": !Ref "AWS::AccountId"
+      Description: !Sub "DO NOT DELETE - Used by Lambda. Created by CloudFormation ${AWS::StackId}"
+      Policies:
+        - PolicyName: OrganizationAccessPolicy
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "cloudformation:ActivateOrganizationsAccess"
+                  - "cloudformation:DeactivateOrganizationsAccess"
+                  - "cloudformation:DescribeOrganizationsAccess"
+                  - "organizations:ListRoots"
+                  - "organizations:EnablePolicyType"
+                  - "organizations:DisablePolicyType"
+                Resource: "*"
+              - Effect: Allow
+                Action:
+                  - "iam:GetRole"
+                  - "iam:DeleteServiceLinkedRole"
+                  - "iam:GetServiceLinkedRoleDeletionStatus"
+                Resource: !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/aws-service-role/stacksets.cloudformation.amazonaws.com/AWSServiceRoleForCloudFormationStackSetsOrgAdmin*"
+              - Effect: Allow
+                Action:
+                  - "organizations:EnableAWSServiceAccess"
+                  - "organizations:DisableAWSServiceAccess"
+                Resource: "*"
+                Condition:
+                  StringEquals:
+                    "organizations:ServicePrincipal": "member.org.stacksets.cloudformation.amazonaws.com"
+              - Effect: Allow
+                Action: "iam:CreateServiceLinkedRole"
+                Resource: !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/aws-service-role/stacksets.cloudformation.amazonaws.com/AWSServiceRoleForCloudFormationStackSetsOrgAdmin*"
+                #Condition:
```

Comment:
This is commented out for now because it doesn't work when uncommented. I have an internal ticket with the CloudFormation team working to figure out why.